### ISSUE TITLE
.gov is available on CZDS

### ISSUE THREAD

**Issue by flotwig** [pzb/TLDs#1]
First off - thank you for this project, it has been very helpful to me.

Within the last year, .gov has become available on the ICANN CZDS, the entire zone can be downloaded from https://czds.icann.org

.arpa, .mil, .int, and .edu are still not available on CZDS.

**Comment by pzb** [comment]
Yes, and .gov makes update available in other ways as well. https://get.gov/about/data/ is a huge improvement over when I built this project.
